app/views/proposals/show_next.html.haml

```diff
@@ -1,3 +1,6 @@
+- if !list_view_conditions && detail_view_conditions
```

Comment:
@phirefly Then this kicks in: https://github.com/18F/C2/blob/master/app/views/layouts/application.html.haml#L17-L18